tools/ci_fetch_deps.py

```diff
@@ -66,14 +66,14 @@ def run(title, command):
 if target == "test":
     submodules = ["extmod/", "lib/", "tools/", "extmod/ulab", "lib/berkeley-db-1.xx"]
 elif target == "docs":
-    submodules = ["extmod/ulab/"]
+    submodules = ["extmod/ulab/", "frozen/"]
 elif target == "mpy-cross-mac":
```

Comment:
Thank you! This addresses my concerns about the CI. I didn't review the PR in other respects, though.